manifests/klusterlet/management/klusterlet-agent-deployment.yaml

```diff
@@ -121,6 +121,10 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.name
+          {{if and .RegistrationDriver .RegistrationDriver.AuthType (eq .RegistrationDriver.AuthType "awsirsa")}}
```

Comment:
seems to me you only need the eq part

Reply:
@qiujian16 We've addressed the comment by only checking the auth type value as suggested.